fixieai/cli/agent/commands.py

```diff
@@ -442,6 +442,10 @@ def _watch_tunnel():
         console.print(
             f"🦊 Agent [green]{agent_api.agent_id}[/] running locally on {host}:{port}, served via {agent_api.func_url}"
         )
+        console.print(
+            f"You can access your agent via [green]https://app.fixie.ai/agents[/]"
```

Comment:
This is a good idea. Can you please link to the actual agent page?
e.g., 
https://app.fixie.ai/agents/mdw/infoviz

Reply:
Good point, updated.